packages/lit-dev-content/site/tutorials/content/custom-attribute-converter/02.md

```diff
@@ -1,38 +1,36 @@
-In `index.html`, `<my-element>`'s `date` property is still being set with Javascript. Requiring Javascript to set a property is not ideal for users of a `<my-element>` because setting properties this way is not as readable or easy as setting attributes.
+In the last step, you added a `dateStr` property that could be set from an attribute, but your component's `date` property is still being set with JavaScript. Requiring JavaScript to set a property is not ideal for users of a `<date-display>` because setting properties this way is not as readable or easy as setting attributes.
 
-## Goal for this step
-
-Accept `date-str` as an attribute in HTML, synchronize the `date` property with `dateStr` property whenever `dateStr` changes, and display the user-defined date in a readable manner using `toLocaleDateString`.
+In this step, you'll add code to synchronize the `date` property with the `dateStr` property whenever `dateStr` changes.
 
 ### Accept `date-str` as an attribute
 
-* Stop passing the `Date` to `date` property of `my-element`
-* Pass `05/05/22` to the `date-str` attribute of `my-element`
+* Stop use JavaScript to set the `date` property of `date-display`.
```

Comment:
> Stop use JavaScript

Typo

Reply:
No Typo. Is Manifesto. STOP USE JAVASCRIPT ELLIOTT

JUST STOP

Reply:
OK I FIX